server/logging.go

```diff
@@ -0,0 +1,74 @@
+package server
```

Comment:
You have created a new `logging.go` with an already existing `logger.go`. This just creates confusion. Let's just delete all the old code, and keep the same file name. i.e.

- Delete all old code in logger.go.
- Move all the new code in that file.
- We can change the signature of `server.NewLogger` to return the mlog.Logger itself, instead of declaring a new function.
- Move the test code to logger_test.go and delete all the old test code for the old logger as well.